stable/envoy/templates/deployment.yaml

```
@@ -42,6 +42,10 @@ spec:
       priorityClassName: "{{ .Values.priorityClassName }}"
 {{- end }}
       terminationGracePeriodSeconds: {{ .Values.terminationGracePeriodSeconds }}
+      {{ if .Values.initContainersTemplate -}}
+      initContainers:
+{{ tpl .Values.initContainersTemplate $ | indent 8 }}
```

For readability, consider using [nindent](http://masterminds.github.io/sprig/strings.html#nindent):

```suggestion
        {{ tpl .Values.initContainersTemplate $ | nindent 8 }}
```
Check out the most recent `helm create` function for an example. 

Could update all instances of this in the file/chart if you want. Or this suggestion could be a follow-up: `indent` is still legit just not quite as readable.

---

Good to know and `nindent` is so nice! Addressed in 9146322